config/plugins.js

```diff
@@ -10,12 +10,13 @@ module.exports = {
 	},
 	// -- end repeat
 	plugins: [
-		'no-use-extend-native',
 		'ava',
-		'unicorn',
-		'promise',
 		'import',
-		'node'
+		'no-use-extend-native',
+		'node',
+		'promise',
+		'typescript',
```

Comment:
This is different than the scope of this PR. If we start using `eslint-plugin-typescript` this should be done in a different PR, that might potentially be released later when the plugin is stable.

Reply:
I disagree. It's pretty usable right now, just missing a lot of rules.

Reply:
"Early stages" does not imply "unstable" or "not production ready". Recent [npm stats](https://npm-stat.com/charts.html?package=eslint-plugin-typescript&package=xo) indicate that users have confidence in its current usability.